ibm/service/iamidentity/resource_ibm_iam_account_settings_template.go

```diff
@@ -94,11 +101,13 @@ func ResourceIBMAccountSettingsTemplate() *schema.Resource {
 						"session_expiration_in_seconds": {
 							Type:        schema.TypeString,
 							Optional:    true,
+							Default:     "86400",
```

Comment:
What is the behaviour earlier when we didn't had the Default value.
Backend was it setting to "86400"

For all three arguments